x-pack/test/cases_api_integration/common/lib/mock.ts

```diff
@@ -48,6 +48,7 @@ export const postCaseReq: CasePostRequest = {
   },
   owner: 'securitySolutionFixture',
   assignees: [],
+  customFields: [],
```

Comment:
I dont know if these tests should be fixed like this. `CasesPostRequest` marks `customFields` as potentially undefined so if some tests were failing I think the problem might be in `x-pack/plugins/cases/server/client/cases/create.ts`.